packages/@aws-cdk-testing/framework-integ/test/aws-ses-actions/test/integ.actions.ts

```diff
@@ -13,14 +13,21 @@ import { STANDARD_NODEJS_RUNTIME } from '../../config';
  * 1. Create a free Workmail test domain (https://us-east-1.console.aws.amazon.com/workmail/v2/home?region=us-east-1#/organizations/create)
  *  - It should automatically be added to your list of verified SES domains, no need to exit the SES sandbox
  * 2. Add a new user email address in the Workmail console
- * 3. Update the TEST_EMAIL constant with the email address of the user you created
- * 4. Deploy the stack with --no-clean, and send an email to the email address you created
- * 5. Check the following:
+ * 3. Remove the automatically created INBOUND_MAIL rule set (https://us-east-1.console.aws.amazon.com/ses/home?region=us-east-1#/email-receiving/INBOUND_MAIL)
```

Comment:
Can you explain why we need this step/change?

Reply:
The default `INBOND_MAIL` is created by WorkMail and already has a WorkMail rule. I was updating the rule before this PR, but we need to test that it's being created correctly now